app/src/main/java/me/ash/reader/ui/page/common/HomeEntry.kt

```diff
@@ -152,13 +157,14 @@ fun HomeEntry(
                 )
             }
             forwardAndBackwardComposable(route = RouteName.FLOW) {
-                FlowPage(
+                FlowRoute(
                     navController = navController,
-                    homeViewModel = homeViewModel,
+                    isExpandedScreen = isExpandedScreen,
+                    homeViewModel = homeViewModel
                 )
             }
             forwardAndBackwardComposable(route = "${RouteName.READING}/{articleId}") {
-                ReadingPage(navController = navController, homeViewModel = homeViewModel)
+                ReadingPage(navController = navController, homeViewModel = homeViewModel, isExpandedScreen = isExpandedScreen)
```

Comment:
TODO: handle system navigations & screen orientation changes

Reply:
@JunkFood02  Could you elaborate on this one? Currently I only added a FlowRoute so that it can react to different width, that include the screen orientation changes.